draft-ietf-acme-acme.md

```diff
@@ -1067,9 +1069,8 @@ field.  This can be used to an ACME account with an existing account in a
 non-ACME system, such as a CA customer database.
 
 To enable ACME account binding, a CA needs to provision the ACME client with a
-MAC key and a key identifier.  The key identifier MUST be an ASCII string.  The
```

Comment:
"The key identifier MUST be an ASCII string" is normative language. Let's leave it in for this "small edits" PR.